### PR TITLE
docs(fx112-relnote): Add support for updated ray() function syntax

### PR DESCRIPTION
**Pull request by dipikabh**
The syntax for the ray() function (value in [offset-path](https://developer.mozilla.org/en-US/docs/Web/CSS/offset-path)) has been updated in Fx112.

- `<ray-size>` is now optional.
- The default value of `<ray-size>` is `closest-side`.

### Related issues

Doc issue tracker: https://github.com/mdn/content/issues/25358
Content PR: https://github.com/mdn/content/pull/26266
Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1820071
Spec: https://drafts.fxtf.org/motion/#ray-function

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks, going to take a look shortly 👀

File: files/en-us/mozilla/firefox/releases/112/index.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This article provides information about the changes in Firefox 112 that affect d
 - [Exponential functions](/en-US/docs/Web/CSS/CSS_Functions#exponential_functions) are now enabled by default.
   This allows the use of `pow()`, `sqrt()`, `hypot()`, `log()` and `exp()` functions ([Firefox bug 1814469](https://bugzil.la/1814469)).
 - The `overlay` keyword value for the {{cssxref("overflow")}} property is now supported as a legacy alias of the keyword value `auto` ([Firefox bug 1817189](https://bugzil.la/1817189)).
+- The updated syntax for the `ray()` function value of the [`offset-path`](/en-US/docs/Web/CSS/offset-path) property is now supported. The `<ray_size>` parameter in the `ray()` function is now optional and has a default value of `closest-side` ([Firefox bug 1820071](https://bugzil.la/1820071)).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/112/index.md:25]: What do you think of something like this?

```suggestion
- The `<ray_size>` parameter is now optional in the `ray()` function that defines an [`offset-path`](/en-US/docs/Web/CSS/offset-path).
If no `<ray_size>` parameter is provided, it has a default value of `closest-side` ([Firefox bug 1820071](https://bugzil.la/1820071)).
```

Comment (estelle) [reply files/en-us/mozilla/firefox/releases/112/index.md:25]: FYI: Safari does not support omitting values. `ray(0deg)` fails in safari TP

**Review by bsmth — APPROVED** [review]
Thanks @dipikabh - merging shortly 👍🏻

**Pull request merged**